peakina/readers/excel.py

```diff
@@ -0,0 +1,199 @@
+"""
+Module to add xml support
```

Comment:
Excel is not generic xml

```suggestion
Module to add excel files support
```